src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/PdfDecorator.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.managerecallsapi.documents
+
+import com.lowagie.text.Element
+import com.lowagie.text.Phrase
+import com.lowagie.text.pdf.ColumnText
+import com.lowagie.text.pdf.PdfReader
+import com.lowagie.text.pdf.PdfStamper
+import org.librepdf.openpdf.fonts.Liberation
+import org.springframework.stereotype.Component
+import java.io.ByteArrayOutputStream
+
+@Component
+class PdfDecorator {
+
+  fun numberPages(pdfContent: ByteArray): ByteArray {
```

Comment:
Given we have no unit test of this code how about the ugly solution of a comment like e.g.:
  // Note: this function is tested within CreateDossierComponentTest

Reply:
But its not tested :) its mocked in that test

Reply:
Where's :mr-hankey: when you need him?

Reply:
OK.  I've added a unit test for `fun numberPages` now: https://github.com/ministryofjustice/manage-recalls-api/pull/165